libs/apriltag/CMakeLists.txt

```diff
@@ -0,0 +1,298 @@
+include(GNUInstallDirs)
+include(GenerateExportHeader)
+include(CMake/apriltagHelperFunctions.cmake)
+
+### Build Apriltag Libraries
+## The following targets are built
+##      apriltag-detector (Apriltag's core functionality with only API functions exported)
+##      apriltag-tags (Apriltag's tag libraries)
+##      apriltag-<tag_family_name> e.g apriltag-tagStandard41h12 (Individual libraries for each tag family)
```

Comment:
Why is there a library containing all tags and individual libraries per tag? Until we can load tags via plugins, I would suggest that one library containing all standard tags is sufficient.